transport/src/main/java/io/netty/channel/DefaultChannelId.java

```diff
@@ -69,16 +69,14 @@ public static DefaultChannelId newInstance() {
             if (processId < 0) {
                 processId = -1;
                 logger.warn("-Dio.netty.processId: {} (malformed)", customProcessId);
-            } else if (logger.isDebugEnabled()) {
-                logger.debug("-Dio.netty.processId: {} (user-set)", processId);
             }
+
+            logger.debug("-Dio.netty.processId: {} (user-set)", processId);
```

Comment:
this should be moved in the else block as otherwise you log a warn and a debug log.